gulpfile.js

```diff
@@ -0,0 +1,127 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview node module that automate tasks.
+ *
+ * @author barnabasmakonda@gmail.com (Barnabas Makonda)
+ */
+
+var argv = require('yargs')
+    .usage(
+      'Usage: $0 --gaeDevserverPath [path to app engine]' +
+      '--clear_datastore [whether to clear all data storage]' +
+      '--enable_sendmail [whether to send emails]' +
+      '--minify [whether to minify third-party dependencies]').argv;
+var concat = require('gulp-concat');
+var gulp = require('gulp');
+var gulpStartGae = require('./scripts/gulp-start-gae-devserver');
+var gulpUtil = require('gulp-util');
+var manifest = require('./manifest.json');
+var minifyCss = require('gulp-minify-css');
+var path = require('path');
+
+var gaeDevserverPath = argv.gaeDevserverPath;
```

Comment:
Ah -- just so you know, this is how I found it: I googled the error message and found this issue, and it hinted at a problem with the first argument, so I started looking into that.

    https://github.com/jonschlinkert/gh-clone/issues/1

One other technique I know of is, if it was working before and it wasn't working now, then the error must have arisen in the commits between then and now. So if you are running a local manual test each time before you commit (which you should!), this gives you tighter bounds for where the error must be located.